package.json

```diff
@@ -42,22 +47,37 @@
             }
         ]
     },
+    "scripts": {
+        "vscode:prepublish": "npm run compile",
+        "compile": "tsc -p ./",
+        "watch": "tsc -watch -p ./",
+        "pack": "vsce package",
+        "postinstall": "node ./node_modules/vscode/bin/install"
+    },
     "bugs": {
         "url": "https://github.com/sodatea/vscode-velocity/issues"
     },
     "homepage": "https://github.com/sodatea/vscode-velocity#readme",
-    "dependencies": {},
-    "devDependencies": {},
+    "dependencies": {
+        "npm": "^6.5.0"
```

Comment:
Why is npm depended on?

Reply:
Removed. This was added here because I was trying update my npm.